### PR TITLE
Create fake mock api routes

### LINKED ISSUE #4: Create mock API routes
## Description

Create mock API routes in order to access our mock data in [pv-sites-mobile/data](https://github.com/openclimatefix/pv-sites-mobile/tree/main/data)

## Todos

- [x] create mock API routes 
- [x] align routes with [pv-site-api](https://github.com/openclimatefix/pv-site-api)

These points are a rough guideline. **Please** feel free to discuss with others on the team about the best way to design this component!

## Useful links

- [pv-site-api](https://github.com/openclimatefix/pv-site-api)
- [mock api route from nowcasting repo](https://github.com/openclimatefix/nowcasting/tree/main/apps/nowcasting-app/pages/api/forecasts)

## Time estimate
tbd!

### PR DESCRIPTION
**Pull request by rohanvan123**
## Description

This feature is mainly for testing purposes. I created an `/api` folder in the `/pages` directory to hold the mock api routes file . This essentially mimics api calls by comparing the inputted routes to certain routes that are similar to the actual API. There are three files with test data, which we displayed as our JSON data for the specifying calls. I also ensured that we handle the cases when an invalid URL is made by throwing a 404.

Fixes #4 
## Todos
- Work on formatting and ensuring the code is up to standards
- Ensure that routes match the style of that of the pv-site-api

## Screenshots

Screenshot 2023-02-16 at 7 49 16 PM
Screenshot 2023-02-16 at 7 50 14 PM

### CODE REVIEW

File: package.json
Comment (AndrewLester) [inline package.json]: We won't use this yet since it'll take some more setup, perhaps down the line we will use it though. So for now `yarn remove @sentry/nextjs`

File: pages/api/[...mockApiRoute].ts
Comment (AndrewLester) [inline pages/api/[...mockApiRoute].ts]:
```suggestion

```

Since we're not using sentry yet.

File: pages/api/[...mockApiRoute].ts
Comment (AndrewLester) [inline pages/api/[...mockApiRoute].ts]: Since these are unused they can be removed
```suggestion

```

File: pages/api/[...mockApiRoute].ts
Comment (AndrewLester) [inline pages/api/[...mockApiRoute].ts]:
```suggestion

```

console log can be removed in production

File: pages/api/[...mockApiRoute].ts
Comment (AndrewLester) [inline pages/api/[...mockApiRoute].ts]: Can be removed since we're not using sentry.
```suggestion
export default handler;
```

**Review by AndrewLester — APPROVED** [review]
Several small changes. Approving this so you can merge after they're fixed. Nice work!

You'll also notice the `format` GitHub check is failing. Just run `yarn format` and commit and push to fix. You can check this locally using `yarn format:check`.

File: pages/api/[...mockApiRoute].ts
Comment (ericcccsliu) [inline pages/api/[...mockApiRoute].ts]: remove unused comments

**Review by ericcccsliu — DISMISSED** [review]
great work Rohan—thanks for getting this up so soon. Just some very mild code quality fixes to be made before this should be merged in. Also, just as a reminder, make sure that your commit messages and pull request titles are written in the [imperative mood!](https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines#:~:text=If%20you%20want%20to%20write,Instead%2C%20describe%20what%20was%20done.) i.e. "create..." rather than "created...". Finally, before merging, make sure that your code passes the prettier format check!

File: pages/api/[...mockApiRoute].ts
Comment (ericcccsliu) [reply pages/api/[...mockApiRoute].ts]: also, in the future, make sure not to resolve comments until the underlying problem is addressed!

**Review by ericcccsliu — CHANGES_REQUESTED** [review]
Since it's early on in the semester, I'll hold off on approving this until it passes our prettier format test. Really great work though—once the format issues are addressed we'll be good to go!!

**Review by AndrewLester — APPROVED** [review]
Great work!

**Review by ericcccsliu — APPROVED** [review]

**Review by NextInvestors — COMMENTED** [review]
Duplicate of #Open Climate Fix (OCF) is a non-profit product lab, totally focused on reducing greenhouse gas emissions as rapidly as possible. Every part of the organisation is designed to maximise climate impact, such as our open and collaborative approach, our rapid prototyping, and our attention on finding scalable & practical solutions.

How to get involved?

At OCF we are passionate that all the knowledge we produce remains open, to break down intellectual property barriers and subsequently reduce the “time to impact”. We therefore encourage external users to use our base code.

There are a number of ways for you to get involved:

Submit pull requests. Right now we are laser-focused on [our solar electricity forecasting project](https://github.com/openclimatefix/nowcasting):

[List of "good first issues"](https://github.com/search?l=&p=1&q=user%3Aopenclimatefix+label%3A%22good+first+issue%22&ref=advsearch&type=Issues&utf8=%E2%9C%93&state=open): GitHub "issues" which describe changes we'd like to make to the code
[OCF's coding style](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
The main tools we use include: PyTorch, PyTorch Lighting, xarray, pandas, pvlib
Sign up to our [newsletter](https://openclimatefix.us20.list-manage.com/subscribe?u=ceb017fe32f0a4620227fda8a&id=d56fc32a31) and follow us on [Twitter](https://twitter.com/OpenClimateFix) and [LinkedIn](https://www.linkedin.com/company/19123036/admin/) to learn the latest about our work

Spread the word with your networks

Use our code(!) by following the guidelines below.

What if you use our code?

In order for us to understand the use of our models, and to demonstrate impact to future funders, it is invaluable for us to know who is using the code and if possible, how. We licence the code in this repository under a permissive MIT licence and if you are using the code or deriving from it, we request that you attribute the use of Open Climate Fix’s work in your product by adding the text below:

'original code by [Open Climate Fix](https://github.com/openclimatefix)'

**Pull request merged**
